docs/installing.md

```diff
@@ -68,6 +68,8 @@ If you don't use *screen*, open 4 terminal windows and run in each one of them:
   1. `make seednode-local`
   2. `make user1-desktop-local` or if you want to run user1 as a daemon: `make user1-daemon-local`
   3. `make user2-desktop-local` or if you want to run user2 as a daemon: `make user2-daemon-local`
+  4. `make arbitrator-desktop-local` or if you want to run arbitrator as a daemon: `make arbitrator-daemon-local`
+  5. Run a local price node [../haveno-pricenode/blob/main/README.md](https://github.com/haveno-dex/haveno-pricenode/blob/main/README.md)
```

Comment:
Recommended change for simplicity:

Optionally [run a local price node](https://github.com/haveno-dex/haveno-pricenode/blob/main/README.md).